### ISSUE TITLE
Yarn Nightly deb repository is not updated consistently

### ISSUE THREAD

**Issue by wolfgang42** [yarnpkg/yarn#7130]
The [nightly builds page](https://yarnpkg.com/lang/en/docs/nightly/) says that the latest version of the build is `yarn_1.16.0-20190319.1101_all.deb`; however, when I try to use the repository according to the instructions the latest version that I get is `yarn_1.13.0-20181121.1436-1_all.deb` which is considerably older. It looks like the nightly repository is broken?

**Comment by Daniel15** [comment]
Since writing the package files to the artifacts directory works fine, maybe I should update the script to read the Debian files directly from the artifacts directory in a smarter way. For example:
1. Look at **all** .deb files in the artifacts directory
2. Check if any of them are missing from the `Packages` file in the repo (https://nightly.yarnpkg.com/debian/dists/nightly/main/binary-amd64/Packages)
3. For each missing package, add that specific one to the repo

**Comment by wolfgang42** [comment]
Aha, yarnpkg/release-infra#24 is the issue I couldn't find.

At the very least, if this isn't working right now there should be a warning on the page to that effect. I was very confused trying to figure out what I'd done wrong to cause apt to pin to such an old version.

**Comment by Daniel15** [comment]
> if this isn't working right now there should be a warning on the page to that effect

Yeah, I definitely need to set up some monitoring for it. I'll try to look into that soon.
